assets/src/components/ladder.tsx

```diff
@@ -285,4 +297,9 @@ function partition<T>(items: T[], testFn: (value: T) => boolean): T[][] {
   )
 }
 
+const statusClass = (
+  headwaySpacing: HeadwaySpacing,
+  status: VehicleAdherenceStatus
+): string => (headwaySpacing === null || status === "off-course" ? status : "")
```

Comment:
I missed this before, but is the `status === "off-course"` check necessary? Would it be just the same if it were `headwaySpacing === null ? status : ""`?

Reply:
no -- i originally had it that way, but then realized that off-track vehicles still get a `headwaySpacing` value so were being colored gray instead of black.

Reply:
Oh I see, I was thinking about it backwards. Thanks.